src/pages/index.tsx

```diff
@@ -128,7 +122,7 @@ function bookDisplay() {
 
   return (
     <div>
-      {data.map((book: Book) => (
+      {data.map((book: any) => (
```

Comment:
so this shouldn't be an Any (we should almost never use any), it should come from Book exported by the `trpc` client

Reply:
plz also fix the merge conflicts

Reply:
thanks for the photo! helpful